### PR TITLE
Support VS2013 build

### PR DESCRIPTION
**Pull request by juanper**

### CODE REVIEW

File: vcbuild.bat [hunk 1]
```diff
@@ -41,7 +41,15 @@ shift
 goto next-arg
 :args-done
 
+@rem Look for Visual Studio 2013
+if not defined VS120COMNTOOLS goto vc-set-2011
+if not exist "%VS120COMNTOOLS%\..\..\vc\vcvarsall.bat" goto vc-set-2011
```
Comment (saghul) [inline vcbuild.bat:46]: Can you name the label vc-set-2012?

**Comment by tjfontaine** [comment]
please see also #1003 which would allow us to shift detection to gyp entirely

**Comment by saghul** [comment]
I haven't tried to compile it myself with VS2013, is that all it's needed? If so, can you please sign the CLA? (http://nodejs.org/cla.html)

**Comment by saghul** [comment]
Sorry, @tjfontaine, I magaed to miss that one, apologies.

**Comment by tjfontaine** [comment]
@saghul no worries

**Comment by rioki** [comment]
I just noticed that we duplicated effort. I filed #1043. If you like I can validate this pull request.

**Comment by saghul** [comment]
Closing since #1043 doesn't need adjustments, in case we merge it. Thanks though!

**Pull request closed without merging**